### PR TITLE
#46 - more dates link

### PR DESCRIPTION
**Pull request by mairead**
[#46](https://github.com/epimorphics/hmlr-linked-data/issues/46)

The issue here was Object.assign called within the JS-routes file passed into the Rails sprockets asset pipeline. IE11 doesn't support Object.assign so the entire JS execution was halted as soon as it throws a console error on Object.assign. 

I added a polyfill to the Sprockets manifest so it would be called above the js-routes to prevent the error from appearing.

### CODE REVIEW

File: public/assets/.sprockets-manifest-4c733f3db87a36dc7d1a215f6e4ea451.json [hunk 1]
```diff
@@ -0,0 +1 @@
..."favicon/android-chrome-512x512-78de3a308300e1dbb21945e2f0d4523890d5bb961333b677dc2acdc4b14f69a0.png","favicon/apple-touch-icon.png":"favicon/apple-touch-icon-256a8a919db049e49179f853fd136cd3b755aaea3770670dd8e17984a9731697.png","favicon/browserconfig.xml":"favicon/browserconfig-37567cb9e7e61d815f9f5b485f2aedff18c93efcd1ba4f96f40999079477b4c9.xml","favicon/mstile-150x150.png":"favicon/mstile-150x150-f7f0ba5f2f7ef5161b1a64e7f3a25b00ef7de5119fee5f7dd179699a945970e8.png","favicon/favicon-16x16.png":"favicon/favicon-16x16-23661f8de3bf76e025c5df9d8c5c7a23429a3527970745467703729c06075791.png","favicon/favicon-32x32.png":"favicon/favicon-32x32-1ff7a3554f5d73ff3721faecfb98c93367d1d01f800d0f80060463081c165ad5.png","favicon/favicon.ico":"favicon/favicon-131d712772e7b6339bc2c7350d543487c5fa4921506f35c9ae4cc4736f9a3878.ico","favicon/manifest.json":"favicon/manifest-aa18db7630d9dbb44c6bc9f03ea69da108b71483ca43c04b8d1e71aa2029dd63.json","favicon/safari-pinned-tab.svg":"favicon/safari-pinned-tab-8e0dde3d60d55c7b5860c5f4a73086a0d57cb7ecd9f29ca703f5b204f0db6dda.svg","lr_common_styles/application.js":"lr_common_styles/application-75487344ef7638b4b78a8b8a4884947ae335d7c13432cce9376cc63c93fa7ca6.js"}}
```
Comment (joescottdave) [inline public/assets/.sprockets-manifest-4c733f3db87a36dc7d1a215f6e4ea451.json]: What is this file?

**Review by joescottdave — CHANGES_REQUESTED** [review]
Looks good. Rubocop is failing though.  

@ijdickinson have hit problems with this Gem before, I think we had to pin it at a lower version to avoid problems.

File: public/assets/.sprockets-manifest-4c733f3db87a36dc7d1a215f6e4ea451.json (hunk 1 above)
Comment (mairead) [reply public/assets/.sprockets-manifest-4c733f3db87a36dc7d1a215f6e4ea451.json]: I think this is autogenerated by sprockets as a manifest of all the things it bundles with rake

**Comment by mairead** [comment]
> Looks good. Rubocop is failing though.
> 
> @ijdickinson have hit problems with this Gem before, I think we had to pin it at a lower version to avoid problems.

rubocop definitely has some strong opinions about your choice of language

**Comment by joescottdave** [comment]
> > Looks good. Rubocop is failing though.
> 
> > 
> 
> > @ijdickinson have hit problems with this Gem before, I think we had to pin it at a lower version to avoid problems.
> 
> 
> 
> rubocop definitely has some strong opinions about your choice of language

Rubyists love the rule of law

**Review by ijdickinson — APPROVED** [review]
Lgtm

**Pull request merged**